setup/README.md

```diff
@@ -35,43 +35,50 @@ gcloud services enable storage-component.googleapis.com
 gcloud services enable ml.googleapis.com
 ```
 
-6. Create and download a service account key with the right permissions, follow the instructions [here](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#iam-service-account-keys-create-gcloud).
+6. Create and download a service account key.
+A [service account](https://cloud.google.com/iam/docs/service-accounts) is a special Google account that belongs to your application or a virtual machine (VM), instead of to an individual end user.
 
-7. Export Environment variables for your Project:
- 
-From your terminal choose from the following two options:
-
-a) Edit `setup.sh` in this directory. Set the following variables:
-    
-   - `PROJECT_ID`
-   - `BUCKET_NAME` 
-   - `GOOGLE_APPLICATION_CREDENTIALS` 
-    
-   Use `source` command to export them:
-  
-   ```bash
-     source ./setup.sh
-   ```
+In order to create a service account key, you may follow the instructions [here](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#iam-service-account-keys-create-gcloud).
+Alternatively, you may run this in your terminal, after setting the values for the 3 environment variables:
 
-b) In your terminal copy and paste the following with the corresponding edits.
 
 ```bash
-export RUNTIME_VERSION=1.13
-export PYTHON_VERSION=3.5
-export REGION=us-central1
+# A name for the service account you are about to create:
+export SERVICE_ACCOUNT_NAME=your-service-account-name
+
+# Your GCP project ID:
+export PROJECT_ID=your-gcp-project-id
+
+# The path where the service account json key should be saved to:
+export GOOGLE_APPLICATION_CREDENTIALS=path/to/service/account/key
 
-# Replace "your-gcp-project-id" with your GCP PROJECT ID
-export PROJECT_ID="your-gcp-project-id"
+#Creathe service account:
+gcloud iam service-accounts create ${MY_SERVICE_ACCOUNT} --display-name="Service Account for ai-platform-samples repo"
 
-# Replace "your-gcp-bucket-name" with a universally unique name for a GCS bucket.
-export BUCKET_NAME="your-gcp-bucket-name"
+# Grant the required roles:
+gcloud projects add-iam-policy-binding endromodal --member serviceAccount:${SERVICE_ACCOUNT_NAME}@${PROJECT_ID}.iam.gserviceaccount.com --role roles/ml.developer
+gcloud projects add-iam-policy-binding endromodal --member serviceAccount:${SERVICE_ACCOUNT_NAME}@${PROJECT_ID}.iam.gserviceaccount.com --role roles/storage.objectAdmin
 
-# Replace "path/to/service/account/key" with the full path to the
```

Comment:
This flow needs to be rearranaged, in item 6 you talk about the credentials and also include the env variables for project/, then in 7 you talk about the env variables.
The way it was it was cleared in terms you have option a or b to setup your env variables, now looks confusing

Reply:
Yeah, I agree. Let me see how I can fix this....